<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Методы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="method">
          <property name="text">
           <string>Метод Гаусса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="method_2">
          <property name="text">
           <string>Метод Крамера</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="method_3">
          <property name="text">
           <string>Метод простой итерации</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="method_4">
          <property name="text">
           <string>Метод Зейделя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="method_5">
          <property name="text">
           <string>Метод верхней релаксации</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Настройка генерируемой матрицы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="prevalence">
          <property name="text">
           <string>Диагональное преобладание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symmetric">
          <property name="text">
           <string>Симметричная</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Размер матрицы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sizeBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Генерация</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changedSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>964</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>939</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>genMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedSize()</slot>
  <slot>genMatrix()</slot>
  <slot>run()</slot>
 </slots>
</ui>
